.github/workflows/cla-assistant.yaml

```diff
@@ -0,0 +1,48 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
+  contents: read # this can be 'read' if the signatures are in remote repository
+  pull-requests: write
+  statuses: write
```

Comment:
This permission would allow arbitrary statuses (including those from other checks, such as the reviewers check) to be set. IMO this permission should be avoided where possible, and the status check should be set based on whether or not the workflow itself errored.

Reply:
Removed `statuses: write` as I don't think it's actually needed. The action does need `actions:read` perms though.